packages/inventory-compliance/src/SystemPolicyCard.js

```diff
@@ -81,15 +81,19 @@ class SystemPolicyCard extends React.Component {
                         </Tooltip> }
                     </TextContent>
                     <div className='margin-bottom-md' >
-                        <Tooltip content={
-                            'The system compliance score is calculated by OpenSCAP and ' +
-                            'is a normalized weighted sum of rules selected for this policy.'
-                        }>
-                            { this.complianceIcon(compliant) }
-                            <Text component={ TextVariants.small }>
-                                { rulesPassed } of { rulesPassed + rulesFailed } rules passed ({ passedPercentage })
-                            </Text>
-                        </Tooltip>
+                        { this.complianceIcon(compliant) }
+                        <Text component={ TextVariants.small }>
+                            { rulesFailed } rule{ rulesFailed == 1 ? '' : 's' } failed
```

Comment:
You could write this as `{ rulesFailed != 1 && 's' }`

Reply:
Indeed. Next time. :wink: